src/booleans.js

```diff
@@ -1,63 +1,123 @@
 function negate(a) {
-  // your code here
+  return !a;
 };
 
 function both(a, b) {
-  // your code here
+  if (a === true && b === true) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function either(a, b) {
-  // your code here
+  if (a === true || b === true) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
As described in the last function, this can be simplified in the same way:
```suggestion
  return a || b;
```